### ISSUE TITLE
Add SqlCommand.Table type when ResultType = ResultType.DataTable

### ISSUE THREAD

**Issue by smoothdeveloper** [fsprojects/FSharp.Data.SqlClient#214]
When using 

```fsharp
type MyCommand = SqlCommandProvider< ..., ResultType = ResultType.DataTable>
```

I'd like to have the DataTable type exported:

```fsharp
let table : MyCommand.Table = ...
```

Right now I can do this:

````fsharp
let table : FSharp.Data.DataTable<MyCommand.Row> = ...
```

but it gives a warning `This API supports the FSharp.Data.SqlClient infrastructure and is not intended to be used directly from your code. `

**Comment by dmitry-a-morozov** [comment]
Agree

**Comment by smoothdeveloper** [comment]
I can contribute this feature if you like some help.

**Comment by dmitry-a-morozov** [comment]
Sure. Give it a shot.
Can you describe a scenario you need it for?

**Comment by smoothdeveloper** [comment]
In interop context with C#, I'm exposing a DataTable property in a record, but I'd like to retain the type in my F# code.

**Comment by dmitry-a-morozov** [comment]
Fixed in v1.8.2
